src/app/features/read-features.js

```
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/mysterion" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const fs = require('fs')
+
+function readFeatures (path) {
+  if (!fs.existsSync(path)) {
+    return null
+  }
+
+  try {
+    return JSON.parse(fs.readFileSync(path).toString())
+  } catch (e) {
+  }
+
+  return null
+}
```

Instead of returning `null`, I would throw error - that way, we can explicitly ignore **local** features in `features.js` and add a better error message for **production** features - i.e. 
```js
try {
  const localFeatures = readFeatures(getFeatureFilePath(LOCAL_FILE))
} catch (err) {
  // local file is optional
}

try {
  const productionFeatures = readFeatures(getFeatureFilePath(PRODUCTION_FILE))
} catch (err) {
  throw new Error(`Failed to read required file: ${PRODUCTION_FILE} : err.toString()`)
}
```